src/homework4/task7.py

```diff
@@ -0,0 +1,14 @@
+"""Даны два натуральных числа. Вычислите их наибольший общий делитель
+   при помощи алгоритма Евклида (мы не знаем функции и рекурсию).
+"""
+
+num_1 = 300
+num_2 = 15
+
+while num_1 and num_2:
+    if num_1 > num_2:
+        num_1 = num_1 % num_2
+    else:
+        num_2 = num_2 % num_1
+
+print(num_1 + num_2)
```

Comment:
It's a cheat - you just don't know where is result (second var is 0).
There is solution without this crutch